internal/pkg/ecs/ecs.go

```diff
@@ -232,6 +247,50 @@ func (c Client) StopDefaultClusterTasks(familyName string) error {
 	return c.ecsClient.StopTasks(taskIDs, ecs.WithStopTaskReason(taskStopReason))
 }
 
+// DecryptedSSMSecrets returns the decrypted parameters from the SSM store.
+func (c Client) DecryptedSSMSecrets(secrets []*ecs.ContainerSecret) ([]EnvVar, error) {
```

Comment:
I think we are fine for now (no action to take)!

I think the `ecs/ecs` package needs a refactor some day though. _(I think)_ ideally we will have these functions ⬇️ 
```go
type TaskDefinition struct {
  ecs.TaskDefinition
}

// TaskDefinition returns the task definition of the service.
func (c Client) TaskDefinition(app, env, svc string) (*TaskDefinition, error)

// EnvVars returns the environment variables of the task definition
func (taskDef *TaskDefinition) EnvVars() ([]EnvVar, error)

type EnvVar interface {
  Name() string
  Value() string
}

type SecretEnvVar interface {
  EnvVar
  ValueFrom() string
}


type plainTextEnvVar struct { // Implements EnvVar.
  name string
  value string
}

func (plain *plainTextEnvVar) Name() string 
func (plain *plainTextEnvVar) Value() string

type ssmEnvVar struct {
  name string
  valueFrom string
} // Implements EnvVar and SecretEnvVar.

// Returns the name of the environment variable whose value is from SSM.
func (ssm *ssmEnvVar) Name() string

// Returns the value of the environment variable.
func (ssm *ssmEnvVar) Value() string

// Returns the name of the SSM parameter where the value is stored.
func (ssm *ssmEnvVar) ValueFrom() string

type secretsManagerEnvVar struct {
    name string
    valueFrom string
} / Implements EnvVar and SecretEnvVar.

func (sm *secretsManagerEnvVar) Name() string
func (sm *secretsManagerEnvVar) Value() string
func (sm *secretsManagerEnvVar) ValueFrom() string



```